Remotion/Core/Tools.UnitTests/AppDomainAssemblyResolverTest.cs

```diff
@@ -165,9 +165,17 @@ private AppDomainAssemblyResolver CreateResolver ()
     private string CreateAssembly (string assemblyName, string moduleName)
     {
       var targetDirectory = AppContext.BaseDirectory;
+#if !NO_PEVERIFY
```

Comment:
Core/Tools gets reverted and will remain as .NET Framework only until we do https://re-motion.atlassian.net/browse/RMTEAM-301